reposerver/repository/repository.go

```diff
@@ -229,6 +231,26 @@ func (s *Service) ListApps(ctx context.Context, q *apiclient.ListAppsRequest) (*
 	return &res, nil
 }
 
+// ListPlugins lists the contents of a GitHub repo
+func (s *Service) ListPlugins(ctx context.Context, _ *empty.Empty) (*apiclient.PluginList, error) {
```

Comment:
what is the use of a blank identifier?

Reply:
That is generated from the proto spec. 
```
rpc ListPlugins(google.protobuf.Empty) returns (PluginList) {
    }
```
List plugins doesn't need any params so we use protobuf.Empty